spec/mongo/server/connection_spec.rb

```diff
@@ -63,7 +67,7 @@ class ConnectionSpecTestException < Exception; end
       before do
         # we want the server to not start in unknown state,
         # hence scan it and transition to some other state here
-        server.scan!
+        #server.scan!
```

Comment:
Is this before block unnecessary now? If so, how do we verify the server was marked unknown, eg on line 200?

Reply:
These blocks are indeed unnecessary. Verifying that the server is marked unknown requires a full server/cluster setup to run sdam flow. The server monitor integration test exercises this path, as well as mongo/monitor spec where this is more explicit.